### ISSUE TITLE
hmacsha256_digest problem ?

### ISSUE THREAD

**Issue by andferrari** [JuliaCloud/AWS.jl#50]
Thank you for this nice package.

When I run `julia launch.jl` I obtain:

```
ERROR: LoadError: MethodError: `hmacsha256_digest` has no method matching hmacsha256_digest(::ASCIIString, ::SubString{ASCIIString})
Closest candidates are:
  hmacsha256_digest(::AbstractString, !Matched::Union{ASCIIString,Array{UInt8,1}})
```

Thank you for your help.

**Comment by s2maki** [comment]
Hello,

Pull request #47 attempts to address this issue.  Try this to see if it resolves your problem.

**Comment by andferrari** [comment]
Yes, ASCIIString solved this problem. Thank you !

**Comment by tanmaykm** [comment]
fixed via #47
